unitTests/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ target_link_libraries(unit_tests
     PRIVATE
         exiv2lib
         ${GTEST_BOTH_LIBRARIES}
+        Threads::Threads
```

Comment:
Note that the IMPORTED target (`GTest::GTest`) will bring the threads dependency automatically. However that IMPORTED target is only available in more recent versions of CMake (3.5 or 3.6).